test/vm.install

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -eu
+
+rpm -e lorax lorax-templates-generic lorax-lmc-virt lorax-lmc-novirt lorax-composer composer-cli || true
+
+# librepo 1.9.6-1 crashes when fetching metadata, which breaks lorax-composer
+sudo dnf update -y librepo
+
+sudo dnf install -y beakerlib /var/tmp/*.rpm
+
+systemctl enable lorax-composer.socket
```

Comment:
Since this is a test we may as well just start up lorax-composer.service instead, may save a small amount of time waiting for the metadata to be fetched.

Reply:
This script is only run once to install the virtual machine. Starting the service at this stage won't make a difference. (The VM is then booted for every test.)

Reply:
If we use lorax-composer.service instead it will start at boot time when it reboots.
Enabling the  .socket is really most useful for systems where you may or many not be using it right away.

Reply:
Ah sorry, I misunderstood (you probably didn't mean that, but wrote "_start_ up..." instead of "enable ..."- systemd nomenclature is weird).

As you say, it will probably only make a small difference, but it's cleaner indeed. Thanks!

Reply:
Please do whatever happens on a production system. It doesn't hurt to test that the socket activation (including the delay on the first call) works as expected.

Reply:
Good point. Left it as is.